components/test/TestAreaRenderer.tsx

```
@@ -0,0 +1,653 @@
+interface TestScenario {
+  name: string;
+  width: number;
+  height: number;
+  scale: number;
+}
+
+interface RenderResult {
+  areaId: string;
+  relative: RelativeCoordinate;
+  absolute: { x: number; y: number; width: number; height: number };
+  validation: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)